libs/auth-api-lib/seeders/data/scope-regulations-admin.ts

```diff
@@ -0,0 +1,30 @@
+import { compose, createScope } from './helpers'
+
+export const up = compose(
+  createScope({
+    name: '@admin.island.is/regulations',
+    displayName: 'Reglugerðir Admin - Create',
+    description: 'Búa til reglugerð',
+    addToResource: '@admin.island.is',
+    addToClients: ['@admin.island.is/web'],
+    accessControlled: true,
+    delegation: {
+      custom: false,
+      legalGuardians: false,
+      procuringHolders: false,
```

Comment:
Both of these should have custom and procuringHolders true.

So we can access control it to a specific government organisation and their procuring holders can then give custom access to it.